package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@copy/v86",
+  "version": "0.1.0",
```

Comment:
Perhaps 0.0.0 or 0.0.1 would make it clearer that v86 is rolling release without backwards-compatibility guarantees.